api/metrics-api-jaxrs/pom.xml

```diff
@@ -177,14 +176,13 @@
           <plugin>
             <groupId>com.github.kongchen</groupId>
             <artifactId>swagger-maven-plugin</artifactId>
-            <version>2.3.3</version>
             <configuration>
               <apiSources>
                 <apiSource>
                   <locations>org.hawkular.metrics.api.jaxrs</locations>
                   <apiVersion>1.0</apiVersion>
                   <basePath>http://localhost:8080/hawkular-metrics/</basePath>
-                  <outputTemplate>${basedir}/src/main/resources/rest-doc/asciidoc.mustache</outputTemplate>
+                  <outputTemplate>https://raw.githubusercontent.com/hawkular/hawkular.github.io/swagger/asciidoc.mustache</outputTemplate>
```

Comment:
btw. it fails on DNS resolving java.net.UnknownHostException, perhaps if you put `raw.githubusercontent.com` to your local hosts file it could respect the 'caching headers'. Everything on 'http://raw.*' has the max-age set only to 300seconds though. The 'caching headers' could be prolonged by using the `rawgit.com` service instead. 

imho it doesn't worth doing